hw/ip/csrng/doc/_index.md

```diff
@@ -0,0 +1,89 @@
+---
+title: "CSRNG HWIP Technical Specification"
+---
+
+# Overview
+
+This document specifies the Cryptographically Secure Random Number Generator (CSRNG) hardware IP functionality.
+Due to the importance of secure random number generation (RNG), it is a topic which is extensively covered in security standards.
+This IP targets compliance with both [BSI’s AIS31 recommendations for Common Criteria](https://www.bsi.bund.de/SharedDocs/Downloads/DE/BSI/Zertifizierung/Interpretationen/AIS_31_Functionality_classes_for_random_number_generators_e.pdf), as well as [NIST's SP 800-90](https://nvlpubs.nist.gov/nistpubs/SpecialPublications/NIST.SP.800-90Ar1.pdf) (as referenced in [FIPS 140-3](https://csrc.nist.gov/publications/detail/fips/140/3/final)).
+Since these two standards use significantly different terminology, it is recommended that the reader refer to our RNG compliance strategy document for an overview of the various RNG classes and equivalencies between the two standards.
+The CSRNG IP supports both of these standards for both deterministic (DRNG) and true random number generation (TRNG).
+In NIST terms, it works with the entropy_source IP to satisfy the requirements as a DRBG or NRBG (non-deterministic random bit generator).
+In AIS31 language, this same implementation satisfies both the DRG.3 requirements for deterministic generation, or the PTG.3 requirements for cryptographically processes physical generation.
+This IP block is attached to the chip interconnect bus as a peripheral module conforming to the [Comportable guideline for peripheral functionality]({{< relref "doc/rm/comportability_specification" >}}), but also has direct hardware links to other IPs for secure internal transmission of random numbers.
+
+## Features
+- Together with a secure entropy source, provides support for both deterministic (DRNG) and true random number generation (TRNG).
```

Comment:
what does secure mean here...? Do we mean it has a certain minimal bar..?

Reply:
There are a couple different meanings:
1. The seeds as they flow from the entropy source to the CSRNG are kept secret.  (seeds are transferred over a secure channel.)
2. There is a guarantee that the entropy level of the seeds meets some minimum entropy level (as defined by both the health checks and whitening implementation)  
Both of these points are discussed in SP 800-90B and AIS31.

Reply:
Hi @tjaychen, I've clarified this in the text. Please see the change in  https://github.com/lowRISC/opentitan/pull/2463/commits/d5f9ec4fc667f9d380a623f4173425a4c509e805